lib/api/treatments/index.js

```diff
@@ -63,18 +63,20 @@ function configure (app, wares, ctx, env) {
       }
     });
 
-    if (!_isNil(d1)) res.setHeader('Last-Modified', d1.toUTCString());
```

Comment:
it is possible that d1 is null, but the next call d1.getTime() was not 'not null safe'.